packages/docmanager/src/widgetmanager.ts

```diff
@@ -252,7 +252,7 @@ export class DocumentWidgetManager implements IDisposable {
    *
    * @param widget - The target widget.
    */
-  protected setCaption(widget: Widget): Promise<void> {
+  protected async setCaption(widget: Widget): Promise<void> {
```

Comment:
This *should* fix the return type from the naked `return` statements below from `untitled` to `Promise<void>`. I assume this is a fix, and not an API break...

Reply:
(i.e. I assume the return type *should* stay the currently typed `Promise<void>`, not the currently implemented `Promise<void> | undefined`.